icon/Icon.js

```diff
@@ -250,6 +250,11 @@ export const Icon = {
     shieldCheck(p)      {return Icon.icon({...p,  iconName: 'shield-check'})},
     sigma(p)            {return Icon.icon({...p,  iconName: 'sigma'})},
     skull(p)            {return Icon.icon({...p,  iconName: 'skull'})},
+    sortArrows(p)       {return Icon.icon({...p, iconName: 'arrow-up-arrow-down'})},
```

Comment:
Please make sure the left brackets are aligned here on the new icons.

Reply:
Left brackets are now aligned.